recipes/constyle/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "constyle" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ceb31c979b2f0ca279aa8b915085405926cf4f0325def271a0e0eb7a2c581096
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - poetry
+  run:
+    - python >=3.7
+
+test:
+  imports:
+    - constyle
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://github.com/abrahammurciano/constyle
```

Comment:
This link is wrong. Should be https://github.com/abrahammurciano/python-constyle